<launch>
    <node name="ros2qnx" pkg="ros2qnx" type="ros2qnx" output="screen" >
        <param name="local_port" value="43894"/>
        <param name="remote_port" value="43893"/>
        <param name="remote_ip" value="192.168.1.120"/>
        <param name="packet_size" value="10"/>

        <param name="goalPose_x" value="0.0"/>
        <param name="goalPose_y" value="0.0"/>
        <param name="goalPose_th" value="0.0"/>
    </node>
</launch>
